apps/jobs/models.py

```diff
@@ -36,8 +40,10 @@ class Submission(TimeStampedModel):
         (FINISHED, FINISHED),
     )
 
-    participant_team = models.ForeignKey(ParticipantTeam, related_name='submissions')
-    challenge_phase = models.ForeignKey(ChallengePhase, related_name='submissions')
+    participant_team = models.ForeignKey(
```

Comment:
can we make this user? That will store the participant team information as well, right? 
It will require one extra join though to query submission per team. Might be good to add both then? 

Saying this because then we will loose out information regarding which user submitted. Not super relevant info but still.